npm/src/controller/api.ts

```diff
@@ -99,7 +118,16 @@ export class APIController implements IAPIController {
    *         description: Unauthorized
    */
   public async config(body: IdPConfig): Promise<OAuth> {
```

Comment:
@niwsa is the `description` mandatory here?

Reply:
Yes `description` is required. Will add it to `_validateIdPConfig`.

Reply:
Will make `description` optional